ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -92,7 +94,7 @@ def failed(self, event: monitoring.CommandFailedEvent):
             span.set_attribute(
                 "db.mongo.duration_micros", event.duration_micros
             )
-            span.set_status(Status(StatusCanonicalCode.UNKNOWN, event.failure))
+            span.set_status(UnknownStatus())
```

Comment:
Same here, `event.failure` could contain valuable information.

Reply:
+1 hiding the actual error would be pretty bad

Reply:
Ok, adding the description field back :+1: